migrations/sql/V1.6__Add_mds_required_documents.sql

```diff
@@ -0,0 +1,102 @@
+CREATE TABLE mine_tailings_storage_facility (
+  mine_tailings_storage_facility_guid uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  mine_tailings_storage_facility_name character varying(60) NOT NULL,
+  mine_guid uuid NOT NULL,
+  create_user      character varying(60) NOT NULL,
+  create_timestamp timestamp with time zone NOT NULL DEFAULT current_timestamp,
+  update_user      character varying(60) NOT NULL,
+  update_timestamp timestamp with time zone NOT NULL DEFAULT current_timestamp,
+
+  FOREIGN KEY (mine_guid) REFERENCES mine_identity(mine_guid) DEFERRABLE INITIALLY DEFERRED
+);
+
+COMMENT ON TABLE mine_tailings_storage_facility IS 'Stores the information relavent to a mine tailings storage facility.';
+
+
+CREATE TABLE mds_required_document_category(
```

Comment:
Sorry, I missed this in the review a while back... can you please look into adding the 4 audit columns (create_user, create_timestamp, update_user, update_timestamp) to this table? This will keep the table consistent with other MDS code/lookup tables. Thanks!